src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Linn.Purchasing.Facade.ResourceBuilders
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Linn.Common.Authorisation;
+    using Linn.Common.Facade;
+    using Linn.Common.Reporting.Models;
+    using Linn.Common.Resources;
+    using Linn.Purchasing.Domain.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+    using Linn.Purchasing.Resources;
+
+    public class PurchaseOrderResourceBuilder : IBuilder<PurchaseOrder>
+    {
+        private readonly IAuthorisationService authService;
+
+        public PurchaseOrderResourceBuilder(IAuthorisationService authService)
+        {
+            this.authService = authService;
+        }
+
+        public PurchaseOrderResource Build(PurchaseOrder entity, IEnumerable<string> claims)
+        {
+            var detail = entity.Details.First(x => x.Line == 1);
```

Comment:
not sure if this detail is ever used but don't hard code anything to line 1 as we might be about to change that